staging/src/k8s.io/cloud-provider/api/well_known_annotations.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apis
+package api
 
 const (
 	// AnnotationProvidedIPAddr is a node IP annotation set by the "external" cloud provider.
```

Comment:
if we're exporting this, make the constant name include Alpha or Beta status, so we can reserve the non-alpha name for the GA annotation (we don't change exported constant values)